index.html

```diff
@@ -0,0 +1,148 @@
+<!DOCTYPE html>
+<html lang="en-GB">
+	<head>
+		<title>Lily Boughton</title>
+		<link rel="stylesheet" href="normalize.css" type="text/css">
+		<link rel="stylesheet" href="styles.css" type="text/css">
+		<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css" integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous">
+	</head>
+	<body>
+		
+		<header>
+			<h1>Hello, I'm <span class="accentColour">Lily Boughton</span>.</h1>
+			<h2>/* I'm hoping to be a full stack developer */</h2>
+			<a href="#nav" class="accentColour">Find out more ></a>
+		</header>
+		
+		<nav id="nav" class="largeNav">
+			<a href="#about">About</a>
+			<a href="#portfolio">Portfolio</a>
+			<a href="#contact">Contact</a>
+		</nav>
+		<nav class="mobileNav">
+			<a href="#">Menu</a> 
+		</nav>
+		
+		<main id="about" class="darkTheme">
+			<h3> &ltabout&gt</h3>
+			<section>
+				<div>
+					<img src="Images/profilePic.jpg" alt="Image of me" class="profilePic"/>
+				</div>
+				<article>
+					<h4>I'm a 'work in progress' hoping to learn the skills to make me an awesome programmer.</h4>
+					<p>I'm Lily. I'm currently studying at <a href="https://io-academy.uk/" target="_blank">iO Academy</a> with the aim to launch a new career in the Coding Community. I've tried my hand at a few things in my working life so far, in each iteration trying to incorporate as much coding as possible!</p>
+					<p>When I'm not bashing away on my keyboard, you can find me out walking my dog, Hendrix, getting creative at my sewing machine or hanging out with my 3 year old nephew.</p>
+				</article>
+			</section>
+		</main>
+		
+		<div id="portfolio" class="lightTheme">
+			<h3>&ltportfolio&gt</h3>
+			<p>Here are some of the projects I have been working on recently ...</p>
+			<section class="tooBig">
```

Comment:
not sure the class name tooBig is very semantic. not entirely sure what the intention is with it